docs/synthetic-tokens/what-are-synthetic-assets.md

```diff
@@ -45,15 +46,25 @@ By using UMA's priceless synthetic token contract template, developers can easil
 
-The EMP allows token sponsors (i.e., people who mint new synthetic tokens) to collateralize their position in a defined collateral currency. Anytime after the EMP’s expiration date, any synthetic tokenholder can redeem their tokens for a settlement value, which is denominated in the collateral currency and fixed to the price of the EMP contract’s price identifier at the expiration timestamp. The price determining the settlement value is resolved by UMA’s oracle contract, the “DVM”. 
+The EMP allows token sponsors (i.e., people who mint new synthetic tokens) to collateralize their position in a defined collateral currency. Anytime after the EMP’s expiration date, any synthetic tokenholder can redeem their tokens for a settlement value, which is denominated in the collateral currency and fixed to the price of the EMP contract’s price identifier at the expiration timestamp. The price determining the settlement value is resolved by UMA’s oracle contract, the “DVM”.
 
 This is only the first example of a synthetic token implementation, and is by no means restrictive on the types of synthetic tokens that could be built on UMA's infrastructure.
 
-View [here](synthetic-tokens/expiring-synthetic-tokens.md) to learn more about UMA's EMP contract. 
+View [here](synthetic-tokens/expiring-synthetic-tokens.md) to learn more about UMA's EMP contract.
+
+### The Perpetual Contract Template
+
+Another contract template approved for use on UMA is the perpetual, which can be used to create synthetic tokens that follow the upward and downward price movement of any asset without an expiration date.
+
+Like the EMP, the perpetual allows token sponsors to collateralize their position in a defined collateral currency and track the price of a different asset through a defined price feed. The synthetic token can be redeemed by token sponsors at any time for its value in the underlying collateral, and the synthetic token's value relative to the underlying collateral is nudged up or down through a funding rate mechanism. By changing the value of the synthetic, the funding rate mechanism effectively pays the token sponsor a fee when the price being tracked goes down, and pays the token holder a fee when the price being tracked goes up.
+
+It is important to note that the price of the synthetic will not precisely mirror the price being tracked, just like the price of a gold ETF does not precisely mirror the price of gold. Through the funding rate mechanism, however, the value of the synthetic should go up or down proportionally alongside the price being tracked.
+
+New funding rates can be proposed through the optimistic oracle, and token sponsor liquidations will take into account both the price being tracked and the cumulative funding rate to determine whether the sponsor has enough collateral. Like liquidations, funding rates can be disputed and settled by UMA token holders through the DVM.
```

Comment:
Thoughts on linking to FTX's funding rate documentation as an example? https://help.ftx.com/hc/en-us/articles/360027946571-Funding